distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/VideoTest.java

```diff
@@ -0,0 +1,304 @@
+
+        final Map<String, Object> streamMonitorProperties = new HashMap<>();
+        streamMonitorProperties.put(UdpStreamMonitor.METATYPE_TITLE, streamTitle);
+        streamMonitorProperties.put(UdpStreamMonitor.METATYPE_MONITORED_ADDRESS, udpStreamAddress);
+        streamMonitorProperties.put(UdpStreamMonitor.METATYPE_METACARD_UPDATE_INITIAL_DELAY, 0);
+        streamMonitorProperties.put(UdpStreamMonitor.METATYPE_BYTE_COUNT_ROLLOVER_CONDITION,
+                5_000_000);
+        streamMonitorProperties.put("startImmediately", true);
+
+        startUdpStreamMonitor(streamMonitorProperties);
+
+        waitForUdpStreamMonitorStart();
+
+        MpegTsUdpClient.broadcastVideo(videoFilePath,
+                LOCALHOST,
+                udpPortNum,
+                NIGHTFLIGHT_DURATION_MS);
+
+        expect("The parent and child metacards to be created").within(10, TimeUnit.SECONDS)
+                .until(() -> executeOpenSearch("xml", "q=*").extract()
+                        .xmlPath()
+                        .getList("metacards.metacard")
+                        .size() == 3);
+
+        final ValidatableResponse parentMetacardResponse = executeOpenSearch("xml",
+                "q=" + streamTitle).log()
+                .all()
+                .assertThat()
+                .body(hasXPath(METACARD_COUNT_XPATH, is("1")))
+                .body(hasXPath("/metacards/metacard/string[@name='title']/value", is(streamTitle)))
+                .body(hasXPath("/metacards/metacard/string[@name='resource-uri']/value",
+                        is(udpStreamAddress)));
+
+        final String parentMetacardId = parentMetacardResponse.extract()
+                .xmlPath()
+                .getString(METACARD_ID_XMLPATH);
+
+        expect("The child metacards to be linked to the parent").within(3, TimeUnit.SECONDS)
+                .until(() -> executeOpenSearch("xml", "q=mpegts-stream*").extract()
+                        .xmlPath()
+                        .getInt("metacards.metacard.string.findAll { it.@name == 'metacard.associations.derived' }.size()")
+                        == 2);
+
+        final String chunkDividerDate = "2009-06-19T07:26:30Z";
+
+        final ValidatableResponse firstChunkMetacardResponse = verifyChunkMetacard(
+                "dtend=" + chunkDividerDate,
+                1212.82825971,
+                "-110.058257 54.791167",
+                parentMetacardId);
+
+        final ValidatableResponse secondChunkMetacardResponse = verifyChunkMetacard(
+                "dtstart=" + chunkDividerDate,
+                1206.75516899,
+                "-110.058421 54.791636",
+                parentMetacardId);
+
+        final long videoFileSize = videoFile.length();
+
+        final long firstChunkLength = getChunkLength(firstChunkMetacardResponse);
+
+        final long secondChunkLength = getChunkLength(secondChunkMetacardResponse);
+
+        assertThat(firstChunkLength + secondChunkLength, is(videoFileSize));
```

Comment:
@jlcsmith @glenhein The test is failing on linux because of this assertion. It passes on my mac, but it seems that on linux some packets are being dropped. Since we're dealing with UDP, there are a number of factors that could cause packets to be dropped. As a result, I'm wondering whether it's even reasonable to test that all the bytes were received.